apps/web/src/services/asset-inventory/components/ServiceAccountAutoSyncForm.vue

```diff
@@ -37,11 +39,88 @@ const emit = defineEmits<{(e:'update:isValid', isValid: boolean): void;
     (e:'change', formData: any): void;
 }>();
 
+const cspAdditionalOptionMap = {
```

Comment:
question: Is this sample? or real data??

Reply:
Sample!